content/joysticks/winwing/winwing-cdu/troubleshooting/_index.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Troubleshooting
+weight: 30
+---
+
+#### I do not want MobiFlight to take over the CDU
+
+Disable it in **Extras->Settings->Peripherals**.
+
+#### A Python message keeps popping up
+
+Install Python and needed packages exactly as described in the [guide](/guides/installing-python/). Or if no MobiFlight support is needed, disable the CDU in **Extras->Settings->Peripherals**.
+
+#### Plane xy does not work
+
+See the list of currently [supported aircraft](/joysticks/winwing/winwing-cdu/supported-aircraft/) and remarks on the planes.
```

Comment:
I don't think this one is necessary.

Reply:
Should not be necessary, but we have so many users asking simple questions, that I thought it does not hurt adding it.

Reply:
Hopefully the way it works is we direct people to the main landing page for the WINWING CDU and they go from there. Since the main page will have the supported planes on it, they'll see that right away without having to dig through troubleshooting info.